src/components/apiDelegation/given/ChooseOrgPage/ChooseOrgPage.tsx

```diff
@@ -6,73 +6,155 @@ import {
   ButtonVariant,
   ButtonColor,
   ButtonSize,
+  SearchField,
+  Panel,
+  PanelVariant,
 } from '@altinn/altinn-design-system';
 import type { Key } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
 import * as React from 'react';
+import { useState, useEffect } from 'react';
 
-import { softAddOrg, softRemoveOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { ActionBar, ActionIconVariant } from '@/components/reusables';
+import {
+  softAddOrg,
+  softRemoveOrg,
+  searchInCurrentOrgs,
+  lookupOrg,
+} from '@/rtk/features/delegableOrg/delegableOrgSlice';
 import { useAppDispatch, useAppSelector } from '@/rtk/app/hooks';
 import type { DelegableOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
 import { ReactComponent as ApiIcon } from '@/assets/ShakeHands.svg';
-import {
-  NewDelegationAccordion,
-  NewDelegationAccordionButtonType,
-} from '@/components/reusables/NewDelegationAccordion';
 import { RouterPath } from '@/routes/Router';
 import { PageContainer } from '@/components/reusables/PageContainer';
 import main from '@/main.module.css';
 
 import classes from './ChooseOrgPage.module.css';
 
 export const ChooseOrgPage = () => {
-  const delegableOrgs = useAppSelector((state) => state.delegableOrg.delegableOrgList);
+  const delegableOrgs = useAppSelector((state) => state.delegableOrg.presentedOrgList);
   const chosenOrgs = useAppSelector((state) => state.delegableOrg.chosenDelegableOrgList);
+  const searchOrgNotExist = useAppSelector((state) => state.delegableOrg.searchOrgNonexistant);
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
+  const [searchString, setSearchString] = useState('');
+  const [promptOrgNumber, setPromptOrgNumber] = useState(false);
+  const fetchLookupOrg = async () => await dispatch(lookupOrg(searchString));
```

Comment:
No need to await dispatch
```suggestion
  const fetchLookupOrg = async () => dispatch(lookupOrg(searchString));
```

Reply:
If I remove it, the lint just adds it again as soon as I hit save 😅
It says: Returning an awaited promise is required in this context